main.html

```diff
@@ -84,17 +86,28 @@
 
   </head>
 
-  <body>
+  <body class="main">
     <div id="output"></div>
 
     <!-- <div id="pageForm" style="display:none;"> -->
-
-    <div id="thumbnail"></div>
-    <!-- <input type="radio" name="addressType" value="Domestic" onchange="GetNumber(this.value)"> -->
-    <button type="button" onClick="updatePageNumber('prev')">Prev</button>
-    <input id="pageNumber" onkeypress="getPageContent()" value="1">
-    <div id="totalPages"></div>
-    <button type="button" onClick="updatePageNumber('next')">Next</button>
+    <div class="source-thumbnails">
+      <div class="change-source">
+        <!-- @Joy the line below should go back to landing.html -->
```

Comment:
left a few @Joy comments for you for things that need some functionality added, but the styles should be right